composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
Maybe we can live with only written instructions. @mibstar let's create a dedicated page in the `/docs` directory for this instead, please.

Thanks!